remove the useless and non-functioning method Zend_Mime_Decode::decodeQuotedPrintable()

Instead, programmers should use the simple (and working) quoted_printable_decode() function.

--- a/library/Zend/Mime/Decode.php
+++ b/library/Zend/Mime/Decode.php
@@ -228,17 +228,4 @@
 
         return $split;
     }
-
-    /**
-     * decode a quoted printable encoded string
-     *
-     * The charset of the returned string depends on your iconv settings.
-     *
-     * @param  string encoded string
-     * @return string decoded string
-     */
-    public static function decodeQuotedPrintable($string)
-    {
-        return iconv_mime_decode($string, ICONV_MIME_DECODE_CONTINUE_ON_ERROR);
-    }
 }
